deploy_github.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -x
```

Comment:
I find this too noisy to be useful usually, but probably can agree to disagree on that

Reply:
I think on longer scripts maybe you're correct. I do like seeing what's happening though in my CI output :smile: